tests/unit/vue-apollo.spec.js

```diff
@@ -0,0 +1,21 @@
+describe('VueApollo', () => {
+  beforeEach(() => {
+    jest.resetModules()
+  })
+
+  it('does not include credentials by default', () => {
+    const { defaultOptions } = require('@/vue-apollo')
+    const { link } = defaultOptions
+
+    expect(link.credentials).toEqual(undefined)
+  })
+
+  it('includes credentials when specifically indicated', () => {
+    process.env.VUE_APP_REQUEST_CREDENTIALS_MODE = 'include'
+
+    const { defaultOptions } = require('@/vue-apollo')
+    const { link } = defaultOptions
+
+    expect(link.credentials).toEqual(undefined)
```

Comment:
this assertion seems incorrect?

Reply:
It is, and I'm working on a proper test case at the moment. I'll let you know once that's done. Thanks!